sycl/test/basic_tests/buffer/subbuffer.cpp

```diff
@@ -0,0 +1,89 @@
+// RUN: %clang -std=c++11 -fsycl %s -o %t.out -lstdc++ -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//==---------- subbuffer.cpp --- sub-buffer basic test ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+
+  bool Failed = false;
+  // Basic test case
+  {
+    const int M = 6;
+    const int N = 7;
+    int Result[M][N] = {0};
+    {
+      auto OrigRange = range<2>(M, N);
```

Comment:
What about using modern C++ `{ }` everywhere instead of old troublesome `()`?
Specially if people look at tests as good coding example and if we consider that SYCL is about modern C++ and heterogeneous computing... :-)